src/utils/utils.ts

```diff
@@ -1 +1,74 @@
-// Write utilities here
+import { Announcement } from "@/models/Announcement";
+
+/**
+ * Filters out announcements whose startDate is in the past.
+ *
+ * @param announcements - The announcements to filter.
```

Comment:
Update doc strings to include the `date`. Make sure to specify that the default value will be the current date.